registries/_format/uri-reference.md

```diff
@@ -0,0 +1,29 @@
+---
+owner: baywet
+issue:
+description: A URI reference as defined in RFC3986
```

Comment:
A link here would be nice (as in several other places)

Reply:
I believe I have added suggestions for all the places that were missing links.  Thanks for the catch.

Reply:
Amazing, thank you Darrel!